src/attention-reporter.js

```diff
@@ -25,18 +25,22 @@
  * @module RS01.attention-reporter
  */
 
-import browser from 'webextension-polyfill';
-import * as Events from "../WebScience/Utilities/Events.js"
-import * as Messaging from "../WebScience/Utilities/Messaging.js"
-import * as PageManager from "../WebScience/Utilities/PageManager.js"
+import browser from "webextension-polyfill";
```

Comment:
It'd be nice to be consistent here (either `'` or `"`), I don't care which :) Would you mind filing an issue to just fix/enforce this w/ eslint? No point cluttering up our patches etc.